wiki/fabrication/fabrication_considerations_for_3D_printing.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Fabrication Considerations for 3D printing
+---
+
+There is more to 3D printing than designing components and parts and replicating those 3D solid models into useful assemblies.  When beginning a design, one must take into account the 3D filament to use, the 3D printing equipment, the model’s final resolution and the limitations of each 3D CAD software to fabricate quality parts.  By keeping these key elements in mind, along with repetition, the 3D printing practitioner and designer will maximize his/her efforts, resulting in high quality finished products.
+
+Following are some of the main consideration one must keep a note of while designing and fabricating parts using 3D printing:
+
+## 1. Wall Thickness
+
+The wall thickness defines the number of times the extruder will lay filament around the perimeter of your model before switching to your infill factor. (Infill is defined separately as a percentage. Typically, 10% or 20% works well for infill.)
+
+Your wall thickness should be defined as a multiple of your nozzle diameter, so a .4mm nozzle with a .8mm wall thickness would result in the printer laying a perimeter two thicknesses wide for each layer. This is the case with the Makerbot Replicator 2.
+
+It is most important to keep this in mind when you are drawing very thin walls. When parts of your model are thin, the perimeter walls are so close together that it doesn’t leave much room for infill in between them. Sometimes, it may not properly fuse the walls to each other, leaving a hollow gap between the walls on the inside of the model.
+
+
+## 2. Part Orientation
+![N|Solid](http://www.ap242.org/image/image_gallery?uuid=a7eb3359-8620-457d-9923-9d4cbec5ba5e&groupId=52520&t=1503324782232)
```

Comment:
Not sure about this, but we'll give it a shot.

Reply:
Tried putting the image into the asset section and then tried accessing it. Didn't work. Any other way of doing it?